### PR TITLE
Fixes #7266: Update documentation to use rudder-server-relay package

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/7266

### CODE REVIEW

File: 4_advanced_usage/75_relay_servers.txt [hunk 1]
```diff
@@ -25,134 +25,22 @@ The procedure will:
...
 
-===== Install Rudder agent
+===== Install Rudder server relay
 
-To begin, please install a regular Rudder agent on the OS.
-
-See <<install-agent, here>> for the complete reference about agent installation.
+To begin, please install a regular Rudder agent on the OS, following the
+<<install-agent, installation instructions>>, but install the 'rudder-server-relay'
+package in addition to the 'rudder-agent' package.
```
Comment (amousset) [inline 4_advanced_usage/75_relay_servers.txt]: rudder-server-relay should be enough but http://www.rudder-project.org/redmine/issues/7593.

File: 4_advanced_usage/75_relay_servers.txt [hunk 2]
```diff
@@ -25,134 +25,22 @@ The procedure will:
...
 ==== Installation procedure - On the relay
 
-===== Install Rudder agent
+===== Install Rudder server relay
 
-To begin, please install a regular Rudder agent on the OS.
-
-See <<install-agent, here>> for the complete reference about agent installation.
+To begin, please install a regular Rudder agent on the OS, following the
+<<install-agent, installation instructions>>, but install the 'rudder-server-relay'
```
Comment (jooooooon) [inline 4_advanced_usage/75_relay_servers.txt]: I find the word "but" surprising here - it leads me to think that you should not do it as indicated in the linked documentation.

How about changing "but" to "and"?

File: 4_advanced_usage/75_relay_servers.txt (hunk 1 above)
Comment (jooooooon) [inline 4_advanced_usage/75_relay_servers.txt]: Argh. Yep, indeed. Let's fix that bug then simplify this documentation afterwards.

File: 4_advanced_usage/75_relay_servers.txt [hunk 3]
```diff
@@ -25,134 +25,22 @@ The procedure will:
...
-
-
 ==== Installation procedure - On the root server
 
 ===== Set the node as a Rudder relay
 
 Launch the rudder-node-to-relay script on the root server, supplying the UUID of the
-host to be added as a relay to it:
+host to be added as a relay to it. You can use the command 'rudder agent info'
+to get the UUID of your node.
```
Comment (jooooooon) [inline 4_advanced_usage/75_relay_servers.txt]: We don't have "rudder agent info" in 3.0.

**Comment by amousset** [comment]
PR updated.

File: 4_advanced_usage/75_relay_servers.txt (hunk 3 above)
Comment (amousset) [inline 4_advanced_usage/75_relay_servers.txt]: Indeed, modified.

Comment (jooooooon) [inline 4_advanced_usage/75_relay_servers.txt]: You could improve this with a PR on rudder-doc on the 3.1 branch though, it would indeed be much simpler!

**Pull request merged**